src/baldr/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ valhalla_module(NAME baldr
     PUBLIC
       ${VALHALLA_SOURCE_DIR}
       ${VALHALLA_SOURCE_DIR}/valhalla
+      ${VALHALLA_SOURCE_DIR}/third_party/rapidjson/include
```

Comment:
It would be good, if possible, to use `-Isystem` on these headers.  They generate a lot of noise when enabling various compiler warnings.